pom.xml

```diff
@@ -907,6 +907,7 @@
               <manifestEntries>
                 <Sealed>true</Sealed>
                 <Implementation-Build>${mvngit.commit.id}</Implementation-Build>
+                <Automatic-Module-Name>org.apache.accumulo</Automatic-Module-Name>
```

Comment:
Will this set this name in every accumulo jar?

Reply:
Yeah all the accumulo-*.jars will have this line:
```Automatic-Module-Name: org.apache.accumulo```

Reply:
`Automatic-Module-Name` is a temporary bridge that allows downstream projects to use the module name we want to have in the future.  Any user facing modules names will be part of our API.  Do you have any long term plans for these API module names?  If so, how does this change help achieve that goal?

Reply:
My thinking was this is the most generic name we could possibly have.  Users who have modules setup can import this module, hopefully using just API code.  Then when we refactor, our internal modules will have more specific names and the API module will have this name.  Ideally this change won't affect semver, since it will only be seen by projects using the module system.  Once we have everything refactored and all our dependencies having stable names, then we can declare this module name as API.

It still doesn't prevent projects from using internal code, but that is no different than what we have now.

Reply:
> Ideally this change won't affect semver, since it will only be seen by projects using the module system.

I think it is tied to semver.  If a downstream project like Fluo creates a module-info.java that requires `org.apache.accumulo` and then Fluo releases, then any changes by Accumulo to that module name would break the released version of Fluo.

Reply:
> > Ideally this change won't affect semver, since it will only be seen by projects using the module system.
> 
> I think it is tied to semver. If a downstream project like Fluo creates a module-info.java that requires `org.apache.accumulo` and then Fluo releases, then any changes by Accumulo to that module name would break the released version of Fluo.

I was thinking of the future changes to the underlying internal packing of the top level module.  But I can see how, even if the name doesn't change, if what is included in the module changes it could break projects using the module.

Reply:
I think each module needs a distinct name.

Reply:
> I think each module needs a distinct name.

This reduces the errors but it is still failing to find the htrace jar.  I think this is because of  "incubating" in the htrace jar name.